contracts/RewardsBooster.sol

```diff
@@ -298,6 +299,8 @@ contract RewardsBooster is Initializable, OwnableUpgradeable, IRewardsBooster, S
         deploymentPool.accRewardsPerBooster = accRewardsPerBooster;
         totalBoosterPoint += _amount;
 
+        require(deploymentPool.boosterPoint >= minimumDeploymentBooster, 'RB015');
```

Comment:
I thought you should check `deploymentPool.accountBooster[_account]`

Reply:
can you add a test to cover this scenaio? when there is boost bigger than min for a deployment and a user tries to add boost smaller than min, it should fail.